factory/processComponents.go

```diff
@@ -15,6 +15,9 @@ import (
 	dataBlock "github.com/ElrondNetwork/elrond-go-core/data/block"
 	"github.com/ElrondNetwork/elrond-go-core/data/indexer"
 	logger "github.com/ElrondNetwork/elrond-go-logger"
+	storageErrorsFactory "github.com/ElrondNetwork/elrond-go-storage/factory"
```

Comment:
```suggestion
	storageFactory "github.com/ElrondNetwork/elrond-go-storage/factory"
```

Reply:
`storageFactory` is already used. renamed to `storageRepoFactory`